<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sistem_Item</class>
 <widget class="QWidget" name="Sistem_Item">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>275</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
background-color: rgb(143, 143, 143)
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="Item_Name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>item name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="Info" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>175</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>175</height>
      </size>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="Firm">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
        </property>
        <property name="text">
         <string>Firm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="Firm_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Seria">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
        </property>
        <property name="text">
         <string>Seria</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="Seria_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Height">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
        </property>
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="Height_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Width">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
        </property>
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="Width_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Lenght">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
        </property>
        <property name="text">
         <string>Lenght</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="Lenght_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="Volume">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
        </property>
        <property name="text">
         <string>Weight</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="Weight_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="Workload">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
        </property>
        <property name="text">
         <string>Workload</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QProgressBar" name="Workload_Bar">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Black</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="Position" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>75</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Black</family>
          <pointsize>12</pointsize>
          <italic>false</italic>
          <fontweight>Black</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(170, 170, 170);</string>
        </property>
        <property name="text">
         <string>Position</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="X">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(170, 170, 170);</string>
          </property>
          <property name="text">
           <string>X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="Y">
          <property name="styleSheet">
           <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(170, 170, 170);</string>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="X_pos">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="Y_pos">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>25</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
